docs/README.md

```diff
@@ -5,9 +5,12 @@ https://flax.readthedocs.io/en/latest/
 
 # How to build the docs
 
-1. Install the requirements using `pip install -r docs/requirements.txt`
-2. Make sure `pandoc` is installed
-3. Run the make script `make html`
+1. Clone the `flax` repository with `git clone https://github.com/google/flax.git`.
+2. In the main `flax` folder, install the required dependencies using `pip install -r docs/requirements.txt`.
+3. Install [`pandoc`](https://pandoc.org): `pip install pandoc`.
+4. [Optional] If you need to make any local changes to the docs, create and switch to a branch. Make your changes to the docs in that branch.
+5. To build the docs, in the `flax/docs` folder run the make script: `make html`.
+6. If the build is successful, you should get the `The HTML pages are in _build/html.` message. You can preview the docs in `flax/docs/_build/html`.
```

Comment:
When writing documentation in VSC, I often run this command in the `flax/docs` directory: ` find ../ ! -regex '.*/[\.|\_].*' | entr -s 'make html'`. This will automatically rebuild the docs on every change you make, which makes it easier to iterate. Note you have to install `entr` for this. Maybe you should add this in step 5?

Reply:
SGTM. The official way to install `entr` is by building it https://github.com/eradman/entr/ first, though Homebrew, apt and others also provide a way (assuming it's not official, since it's not mentioned on the project page). So I'm adding this as an "`[Optional]`" step.

LMKWYT @marcvanzee 

```diff
5. To build the docs.... `make html`.

+  [Optional] Alternatively, install [`entr](https://github.com/eradman/entr/)
+ which helps run arbitrary commands when files change. Then run
+ `find ../ ! -regex '.*/[\.|\_].*' | entr -s 'make html'`.
```

Reply:
Looks great!